src/raw_client.rs

```diff
@@ -277,19 +280,23 @@ impl RawClient<ElectrumSslStream> {
 ))]
 mod danger {
     use rustls;
-    use webpki;
+    use rustls::client::ServerCertVerified;
+    use rustls::{Certificate, Error, ServerName};
+    use std::time::SystemTime;
 
     pub struct NoCertificateVerification {}
 
-    impl rustls::ServerCertVerifier for NoCertificateVerification {
+    impl rustls::client::ServerCertVerifier for NoCertificateVerification {
         fn verify_server_cert(
             &self,
-            _roots: &rustls::RootCertStore,
-            _presented_certs: &[rustls::Certificate],
-            _dns_name: webpki::DNSNameRef<'_>,
-            _ocsp: &[u8],
-        ) -> Result<rustls::ServerCertVerified, rustls::TLSError> {
-            Ok(rustls::ServerCertVerified::assertion())
+            end_entity: &Certificate,
+            intermediates: &[Certificate],
+            server_name: &ServerName,
+            scts: &mut dyn Iterator<Item = &[u8]>,
+            ocsp_response: &[u8],
+            now: SystemTime,
```

Comment:
You should prefix these arguments with the underscore since they are unused

Reply:
Can do!

If we are about this then I think CI should catch such stuff :)

Reply:
I later noticed that there are a few other warnings, so yes we should consider checking for them in the CI. In this case I pointed it out because we were using the underscores before and I saw that you removed them.